entrypoint.sh

```diff
@@ -66,6 +68,14 @@ force_wsgi_environ = ${FF_SYNCSERVER_FORCE_WSGI_ENVIRON}
 #[browserid]
 #backend = tokenserver.verifiers.LocalVerifier
 #audiences = https://localhost:5000
+
 EOL
 
+if [ -n "$FF_SYNCSERVER_FORWARDED_ALLOW_IPS" ]; then
+  cat >> "$SYNCSERVER_INI_PATH" <<EOL
+# If you are running Nginx on a different host than the ff sync server the ff snyc server have to trust the X-Forwarded-* headers sent by Nginx.
+forwarded_allow_ips = $FF_SYNCSERVER_FORWARDED_ALLOW_IPS
```

Comment:
@b00bob Use this format to include variables : `${FF_SYNCSERVER_FORWARDED_ALLOW_IPS}`

Reply:
Yes, that's right. Thanks for the review :+1: